pypeit/core/meta.py

```diff
@@ -132,6 +132,7 @@ def define_additional_meta(nlamps=20):
                        'calpos': dict(dtype=str, comment='Position of calibration system (KCWI)'),
                        'datasec': dict(dtype=str, comment='Data section (windowing)'),
                        'dateobs': dict(dtype=str, comment='Observation date'),
+                       'decker_basename': dict(dtype=str, comment='Slit/mask/decker basename'),
```

Comment:
Can you explain why we need both decker and decker_basename?

Reply:
I added some comments in the code, but basically `decker_basename` is necessary for the reduction of `LONGSLIT` masks that use calibrations taken with a different  `decker`. I could define `decker` in the same way I defined `decker_basename` and get rid of `decker_basename`, but I believe it's important to propagate the information on the full name of the slitmask (`decker`), which is not preserved in `decker_basename` for `LONGSLIT` masks.

Reply:
Okay fair enough. I think people are going to get confused by the naming though which is so similar to decker.  Should we think of something better than decker basename that more clearly indicates when decker_basename is going to be used? Maybe decker_calibration or decker_secondary or decker_calib_secondary and then in the comment you need to clearly spell out how this differs from decker.